src/components/Layout.tsx

```diff
@@ -1,32 +1,111 @@
-import { Box, Container, Heading, HStack } from "@chakra-ui/react"
+import {
+  useColorMode,
+  Switch,
+  Icon,
+  Box,
+  Container,
+  Heading,
+  Stack,
+  HStack,
+} from "@chakra-ui/react"
+import { Moon, Sun } from "phosphor-react"
 import Head from "next/head"
 import Account from "components/web3Connection/Account"
 
 type Props = {
   title: string
   bg?: string
+  bgColor?: string
```

Comment:
I'm not sure if we need both `bg` and `bgColor` here, but I left it in here, so we can decide later.

Reply:
Actually I think we don't need any bg props. We'll only need a theme prop, but we'll only have to deal with that when we'll be implementing the different themes, now it's like it's always "subtle". For colors it should just use the primary palette, for which we should set primary to gray by default in colors.ts